### ISSUE TITLE
Fix issue with league/csv

### ISSUE THREAD

**Issue by fiedsch** [fiedsch/datamanagement#3]
We currently need to lock league/csv to version 9.9.0 in order not to run into errors when calling 

```
while (($line = $data_reader->getLine(Reader::SKIP_EMPTY_LINES)) !== null) {
```

With a too recent version of league/csv we get

```
PHP Fatal error:  Uncaught TypeError: League\Csv\ResultSet::League\Csv\{closure}(): Argument #1 ($record) must be of type array, null given
```

**Comment by fiedsch** [comment]
"Fixed" in 1.2.1 by adding a `conflict` with `"league/csv": ">=9.12"`.

Looks like we need to rework our concept in `CsvReader::getLine()`.

**Comment by fiedsch** [comment]
Fixed in 1.3.0
